src/lib.rs

```diff
@@ -134,6 +135,124 @@ mod util_tests {
     }
 }
 
+/// cec_audio_status which does not allow CEC_AUDIO_VOLUME_STATUS_UNKNOWN
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct KnownCecAudioStatus(u8);
+
+impl KnownCecAudioStatus {
+    pub fn new(volume: u8, is_muted: bool) -> Self {
+        let volume = min(volume, libcec_sys::CEC_AUDIO_VOLUME_MAX as u8);
+        Self(if is_muted {
+            volume | libcec_sys::CEC_AUDIO_MUTE_STATUS_MASK as u8
+        } else {
+            volume
+        })
+    }
+
+    pub fn volume(self) -> u8 {
+        self.0 & libcec_sys::CEC_AUDIO_VOLUME_STATUS_MASK as u8
```

Comment:
This is probably just me but I am having reading the precedence of operations

for me this verbose version is easier on the eyes. What do you think? does lint accept ir?

```suggestion
        self.0 & (libcec_sys::CEC_AUDIO_VOLUME_STATUS_MASK as u8)
```

Reply:
Oh yep! I can add parenthesis. I'm just used to knowing that casts have higher precedence than all other binary expressions: https://doc.rust-lang.org/reference/expressions.html